plugins/scaffolder-field-extensions/scaffolder-frontend-module-http-request-field/README.md

```diff
@@ -0,0 +1,47 @@
+# scaffolder-frontend-module-http-request-field
+
+This custom scaffolder field, fetches an api call from the backstage backend and allows the result to be
+rendered to a list.
+
+It can be installed as follows in the `App.tsx`
+
+```typescript jsx
+...
+    <Route path="/create" element={<ScaffolderPage />}>
+      <ScaffolderFieldExtensions>
+        ...
+        <SelectFieldFromApiExtension />
+      </ScaffolderFieldExtensions>
+    </Route>
+...
+```
+
+Here is an example of its use in a template:
+
+```yaml
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: custom-field-demo
+  title: Custom Field Demo
+  description: Custom Field Demo.
+spec:
+  owner: backstage/techdocs-core
+  type: service
+
+  parameters:
+    properties:
+    - title: Custom
+      properties:
+        custom:
+          title: custom
+          type: string
+          description: My custom name for the component
+          ui:field: SelectFieldFromApi
+          ui:path: "catalog/entity-facets"
+          ui:arraySelector: ['facets', 'kind']
+          ui:valueSelector: 'count'
+          ui:labelSelector: 'value'
+          ui:params:
+            facet: "kind"
```

Comment:
Can we be more specific how can it be used? what does each field mean ? Or if these are some backstage related things lets linkn to the official backstage docs to learn more about.

Reply:
This params is a list of path parameters for the api call. The property probably could probably be named better.

Reply:
I think it would be great if we would add an explanation to these configs, something like:

With these configs the outgoing request will be this:
`https://my.backstage.com/api/catalog/entity-facets?facet=kind`
The response is the following:
```
{
  "facets": [...],
  "kind": [{count: 5, value: 'foo'}]
}
```

Reply:
done. thanks